pvlib/irradiance.py

```diff
@@ -584,23 +579,25 @@ def get_ground_diffuse(surface_tilt, ghi, albedo=.25, surface_type=None):
     Returns
     -------
     grounddiffuse : numeric
-        Ground reflected irradiance. [W/m^2]
+        Ground reflected irradiance. :math:`W/m^2`
 
+    Notes
+    -----
+    Ground diffuse irradiance is the the last term of equations 3, 4, 7, 8,
+    10, 11, and 12 in [1]_. Table of albedo values by ``surface_type``
```

Comment:
It might be more helpful to describe what that last term is, rather than refer to a paper that's not openly accessible.

Reply:
There's a preprint [here](https://strathprints.strath.ac.uk/5008/1/Strachan_PA_et_al_Pure_Empirical_validation_of_models_to_compute_solar_irradiance_on_inclined_surfaces_for_building_energy_simulation_2007.pdf) but one has to find it by searching the title.